.circleci/config.yml

```diff
@@ -544,14 +544,16 @@ commands:
     steps:
       - run: python webui/tests/bin/createUserAndExperiments.py
       - run: |
-          mkdir -p webui/tests/reports/screencasts
+          mkdir -p webui/tests/reports/logs
+          mkdir -p webui/tests/reports/videos
           docker run \
             --name e2e_gauge \
             --network=determined_default \
             --mount type=bind,source=$(pwd)/webui,target=/webui \
             -w /webui/tests \
             --env DET_MASTER=determined-master:8080 \
             determinedai/gauge-taiko-ts
+          rm -f webui/tests/reports/videos/*.jpeg
```

Comment:
suggestion: could we add this step as a post process to the python script handling the test so that it also happens when you run the test locally?

Reply:
ultimately the script approach (e2e-tests.py) couldn't property remove the files via CI due to permission reasons, I left this in here since it's the only way that reliably removes it within the CI.